IMPLEMENTATION_STATUS.md

```diff
@@ -0,0 +1,588 @@
+# IPFS Implementation Status
+
+> Legend: :green_apple: Done &nbsp; :lemon: In Progress &nbsp; :tomato: Missing &nbsp; :chestnut: Not planned
+
+# Table of Contents
+
+# API
+
+## Bitswap
+
+#### CLI
+
+#### HTTP
+
+| Endpoint                                     | Go Impl       | JS Impl       |
+| -------------------------------------------- | :-----------: | :-----------: |
+| **`GET /api/v0/bitswap/stat`**               | :green_apple: | :green_apple: |
+| **`GET /api/v0/bitswap/unwant`**             | :green_apple: | :tomato:      |
+|     `arg=`                                   | :green_apple: | :tomato:      |
+| **`GET /api/v0/bitswap/wantlist`**           | :green_apple: | :lemon:       |
+|     `peer=`                                  | :green_apple: | :tomato:      |
+
+#### Core
+
+
+--------------------------------------------------------------------------------
+
+## Block `ipfs block`
+
+#### CLI
+
+#### HTTP
+
+| Endpoint                                     | Go Impl       | JS Impl       |
+| -------------------------------------------- | :-----------: | :-----------: |
+| **`GET /api/v0/block/get`**                  | :green_apple: | :green_apple: |
+|     `arg=`                                   | :green_apple: | :green_apple: |
+| **`POST /api/v0/block/put`**                 | :green_apple: | :green_apple: |
+|     `arg=`                                   | :green_apple: | :green_apple: |
+| **`GET /api/v0/block/stat`**                 | :green_apple: | :green_apple: |
+|     `arg=`                                   | :green_apple: | :green_apple: |
+
+#### Core
+
+
+--------------------------------------------------------------------------------
+
+## Bootstrap
+
+#### CLI
+
+#### HTTP
+
+| Endpoint                                     | Go Impl       | JS Impl       |
+| -------------------------------------------- | :-----------: | :-----------: |
+| [`GET /api/v0/bootstrap/add`]                | :green_apple: | :green_apple: |
+| [`GET /api/v0/bootstrap/list`]               | :green_apple: | :green_apple: |
+| [`GET /api/v0/bootstrap/rm`]                 | :green_apple: | :green_apple: |
```

Comment:
@Kubuxu see above _"WIP"_ means that I'm not finished. The Apiary documentation was out of date